Automattic/wp-calypso
# Manage all domains: Remove footer upsell carousel

Fixes https://github.com/Automattic/dotcom-forge/issues/3009

## Proposed Changes

* Reverts https://github.com/Automattic/wp-calypso/pull/60882 except for the general carousel improvement.
* See the issue for more info.

## Testing Instructions



1. You need a domain-only subscription to test this PR
2. Build this branch locally or open the live Calypso link
3. Navigate to to /manage/domains
4. Verify that the carousel does not show

## Pre-merge Checklist
